File: SDKs/twilio.com/twilio_serverless_v1/1.36.0/go/pkg/models/operations/deletevariable.go
package operations

import (
"openapi/pkg/models/shared")
var DeleteVariableServers = []string{
	"https://serverless.twilio.com",
}

type DeleteVariablePathParams struct {
    EnvironmentSid string `pathParam:"style=simple,explode=false,name=EnvironmentSid"`
    ServiceSid string `pathParam:"style=simple,explode=false,name=ServiceSid"`
    Sid string `pathParam:"style=simple,explode=false,name=Sid"`
    
}

type DeleteVariableSecurity struct {
    AccountSidAuthToken shared.SchemeAccountSidAuthToken `security:"scheme,type=http,subtype=basic"`
    
}

type DeleteVariableRequest struct {
    ServerURL *string 
    PathParams DeleteVariablePathParams 
    Security DeleteVariableSecurity 
    
}

type DeleteVariableResponse struct {
    ContentType string 
    StatusCode int64 
    
}

File: SDKs/twilio.com/twilio_serverless_v1/1.36.0/typescript/src/sdk/models/operations/deletevariable.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";
import * as shared from "../shared";
export const DELETEVARIABLE_SERVERS = [
	"https://serverless.twilio.com",
];



export class DeleteVariablePathParams extends SpeakeasyBase {
  @Metadata({ data: "pathParam, style=simple;explode=false;name=EnvironmentSid" })
  environmentSid: string;

  @Metadata({ data: "pathParam, style=simple;explode=false;name=ServiceSid" })
  serviceSid: string;

  @Metadata({ data: "pathParam, style=simple;explode=false;name=Sid" })
  sid: string;
}


export class DeleteVariableSecurity extends SpeakeasyBase {
  @Metadata({ data: "security, scheme=true;type=http;subtype=basic" })
  accountSidAuthToken: shared.SchemeAccountSidAuthToken;
}


export class DeleteVariableRequest extends SpeakeasyBase {
  @Metadata()
  serverUrl?: string;

  @Metadata()
  pathParams: DeleteVariablePathParams;

  @Metadata()
  security: DeleteVariableSecurity;
}


export class DeleteVariableResponse extends SpeakeasyBase {
  @Metadata()
  contentType: string;

  @Metadata()
  statusCode: number;
}
